java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/BlockingQueueProducerProvider.java

```diff
@@ -78,4 +83,14 @@ private boolean shouldConsumerSeekToLatest(AzureEventHubsConfig azureEventHubsCo
     throw new ConfigException(AzureEventHubsConfigConstants.CONSUMER_OFFSET, seekValue,
         CONSUMER_OFFSET_EXCEPTION_MESSAGE);
   }
+
+  private static KeyValueTypes getSourceDataTypes(Kcql parsedKcql) {
+    Map<String, String> kcqlProperties = parsedKcql.getProperties();
+    String keyFormat = kcqlProperties.getOrDefault("key.format", SourceDataType.BYTE.name());
```

Comment:
it needs to be bytes not byte

Reply:
Changed! 

I initially thought `BYTE` (singular) would be better cause we can have `AVRO` not `AVROS` in config.